setup.py

```diff
@@ -261,6 +261,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
 samba = ['pysmbclient>=0.1.3']
 segment = ['analytics-python>=1.2.9']
 sendgrid = ['sendgrid>=5.2.0,<6']
+sentry = ['sentry-sdk>=0.8.0', "blinker>=1.1"]
```

Comment:
I don't see blinker used anywhere?

Reply:
Flask has a signals system that the SDK uses to hook into Flask apps. The signalling system requires blinker. It's an optional dependency of Flask.